sass/layout/_navigation.scss

```diff
@@ -35,6 +35,60 @@ a {
  */
 
 /* Small menu. */
+.menu-toggle {
+	cursor: pointer;
+
+	.mobile-nav-toggle-style-button & {
+		padding: 15px 20px;
+		margin: 20px 20px 0;
+		border-radius: 5px;
+		background-color: $accent-blue;
+		color: #fff;
+		font-weight: 400;
+		text: {
+			align: center;
+			transform: uppercase;
+		}
+
+		&:hover {
+			background-color: darken($accent-blue, 20%);
+		}
+	}
+
+	.mobile-nav-toggle-style-hamburger & {
+		float: right;
+		margin: 45px;
+		color: $accent-blue;
+
+		&::before {
+			font: {
+				family: FontAwesome;
+				size: em(30);
+			}
+			content: "\f0c9";
+		}
+
+		span {
+			display: none;
+		}
+	}
+
+	.mobile-nav-toggle-style-hamburger .main-navigation.toggled &::before {
+		font: {
+			family: FontAwesome;
+			size: em(36);
+		}
+		content: "\f00d";
+	}
+}
+
+.toggled {
```

Comment:
I think this selected might need to be nested in another selector if it isn't already. Classes like this are common for plugins to use, and although I love the simplicity I think it's better to be a bit more specific for our use cases to avoid potential conflicts.